### ISSUE TITLE
Allow downloading audio_only

### ISSUE THREAD

**Issue by ihabunek** [ihabunek/twitch-dl#10]

**Comment by miff2000** [comment]
The way I've bene doing this until now is to take the outputted MKV file and run this command against it

```
ffmpeg -i twitch-dl-created.mkv -acodec copy -vn audio-only-file.m4a
```

This produces `audio-only-file.m4a` which I then tag using `mutagen` like this

```python
from mutagen.mp4 import MP4, MP4Cover

video = MP4("audio-only-file.m4a")
video.delete()
video["\xa9nam"] = "Audio track from Twitch video"
video["\xa9ART"] = "Twitch artist"
video["\xa9alb"] = "Twitch Streams"
video["cpil"] = True
video.save()
```

The tag reference info is here: <https://mutagen.readthedocs.io/en/latest/api/mp4.html#mutagen.mp4.MP4Tags>

**Comment by ihabunek** [comment]
Implemented in 1.19.0. Use `--quality audio_only`.
